addons/docs/src/mdx/mdx-compiler-plugin.js

```diff
@@ -310,7 +309,7 @@ function extractExports(node, options) {
   );
 
   const fullJsx = [
-    'import { DocsContainer, makeStoryFn } from "@storybook/addon-docs/blocks";',
+    'import { DocsContainer, makeStoryFn, AddContext } from "@storybook/addon-docs/blocks";',
```

Comment:
Can you remove `DocsContainer` now that you aren't using it in the `page` option? I did a cursory search in this file and didn't see it used anywhere else.

Reply:
good catch! i think that actually solves another problem i was having

Reply:
Nice. Glad to hear it.